### PR TITLE
corrected the endpoint for getting the details of a file

### PR DESCRIPTION
**Pull request by Bernadettechukwuedo**
Endpoint for viewing the details of a file 
linear : https://linear.app/team-socket/issue/BAC-53/endpoint-for-viewing-the-details-of-a-particular-file
I used the get method to view the details of a particular file
Method: Get
path : /org/{org_id}/rooms/{room_id}/messages/{message_id}

### CODE REVIEW

File: backend/endpoints/detail_file.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		room_id: room id number
+        message_id: message id number
+		return:
+			The details of a file uploaded
+
+    org_id :6373eb474746182adae97314
+    6373eb4f4746182adae97316
+    room_id: 6373eb4f4746182adae97316
+    message_id:637fe320601ce3fc5dc739ea
+"""
```
Comment (zxenonx) [inline backend/endpoints/detail_file.py]: That's not where the docstring should be written.
Kindly check the other functions in the codebase.

File: backend/endpoints/detail_file.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    
+    #for file in files:
+    
+  
+    
+    #print(file)
+    #print(detail)
+   
+
```
Comment (zxenonx) [inline backend/endpoints/detail_file.py]: Kindly remove.

File: backend/endpoints/detail_file.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        file_name_split = file.split("/")
+        file_name = file_info[0]
+        file_type = file_info[1]
+        details_of_file = {
+                    "filename" : file_name,
+                    "file-type" : file_type,
+                    "file-size" : file_size
+            }
+        details.append(details_of_file)
+        print(details)
```
Comment (zxenonx) [inline backend/endpoints/detail_file.py]: Remove all the print statements

File: backend/requirements.txt [hunk 4]
```diff
@@ -18,7 +18,11 @@ fastapi-pagination==0.9.1
 filelock==3.3.1
 h11==0.12.0
 httpcore==0.15.0
+<<<<<<< HEAD
+httptools==0.5.0
+=======
 httptools
+>>>>>>> dev
```
Comment (zxenonx) [inline backend/requirements.txt]: Kindly remove the conflicts tags

**Review by zxenonx — DISMISSED** [review]
Good work.
I left some comments.

**Comment by zxenonx** [comment]
> Endpoint for viewing the details of a file

Kindly check the other endpoints to see how they describe their PRs
@Bernadettechukwuedo

File: backend/endpoints/detail_file.py (hunk 1 above)
Comment (Bernadettechukwuedo) [reply backend/endpoints/detail_file.py]: Done

File: backend/endpoints/detail_file.py (hunk 2 above)
Comment (Bernadettechukwuedo) [reply backend/endpoints/detail_file.py]: Done

File: backend/endpoints/detail_file.py (hunk 3 above)
Comment (Bernadettechukwuedo) [reply backend/endpoints/detail_file.py]: Done

File: backend/requirements.txt (hunk 4 above)
Comment (Bernadettechukwuedo) [reply backend/requirements.txt]: Done

File: backend/endpoints/detail_file.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            status_code=status.HTTP_404_NOT_FOUND, detail="file not found"
+        )
+    
+        
+
+    
+
+                
+
+
```
Comment (mikengr) [inline backend/endpoints/detail_file.py:48]: remove the unnecessary extra spaces. One line is enough after an endpoint or function

File: backend/endpoints/detail_file.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        file_name_split = file.split("/")
+        file_name = file_info[0]
+        file_type = file_info[1]
+        details_of_file = {
+                    "filename" : file_name,
+                    "file-type" : file_type,
+                    "file-size" : file_size
+            }
+        details.append(details_of_file)
+        return details
```
Comment (mikengr) [inline backend/endpoints/detail_file.py]: your loop function will return after the first iteration here, is that intended?

**Review by mikengr — COMMENTED** [review]
@Bernadettechukwuedo I left some comments

File: backend/endpoints/detail_file.py (hunk 6 above)
Comment (Bernadettechukwuedo) [reply backend/endpoints/detail_file.py]: Yes

File: backend/endpoints/detail_file.py (hunk 5 above)
Comment (Bernadettechukwuedo) [reply backend/endpoints/detail_file.py:48]: Done

**Review by mikengr — APPROVED** [review]
Good work @Bernadettechukwuedo

**Pull request merged**